packages/admin-ui/ui/src/components/templates/product-table/use-product-column.tsx

```diff
@@ -58,30 +81,33 @@ const useProductTableColumn = ({ setTileView, setListView, showList }) => {
         },
       },
       {
-        Header: "Collection",
+        Header: t("product-table-collection", "Collection"),
         accessor: "collection", // accessor is the "key" in the data
         Cell: ({ cell: { value } }) => {
           return <div>{value?.title || "-"}</div>
         },
       },
       {
-        Header: "Status",
+        Header: t("product-table-status", "Status"),
         accessor: "status",
         Cell: ({ cell: { value } }) => getProductStatus(value),
       },
       {
-        Header: "Availability",
+        Header: t("product-table-availability", "Availability"),
         accessor: "sales_channels",
         Cell: ({ cell: { value } }) => getProductSalesChannels(value),
       },
       {
-        Header: "Inventory",
+        Header: t("product-table-inventory", "Inventory"),
         accessor: "variants",
         Cell: ({ cell: { value } }) => (
           <div>
             {value.reduce((acc, next) => acc + next.inventory_quantity, 0)}
-            {" in stock for "}
-            {value.length} variant(s)
+            {t(
+              "product-table-inventory-in-stock-count",
+              "in stock for {{count}} variant(s)",
```

Comment:
todo: add space in default string + translation
```suggestion
              " in stock for {{count}} variant(s)",
```